sdk/storage/azure-storage-blob-changefeed/azure/storage/blobchangefeed/_models.py

```diff
@@ -0,0 +1,399 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+# pylint: disable=too-few-public-methods, too-many-instance-attributes
+# pylint: disable=super-init-not-called, too-many-lines
+import collections
+import json
+from datetime import datetime
+from io import BytesIO
+
+from azure.storage.blob._shared.avro.datafile import DataFileReader
+from azure.storage.blob._shared.avro.avro_io import DatumReader
+
+from azure.core.exceptions import HttpResponseError
+from azure.core.paging import PageIterator
+
+# ===============================================================================================
+SEGMENT_COMMON_PATH = "idx/segments/"
+PATH_DELIMITER = "/"
+# ===============================================================================================
+
+
+class ChangeFeedPaged(PageIterator):
+    """An Iterable of change feed events
+
+    :ivar int results_per_page:
+        The maximum number of results retrieved per API call.
+    :ivar dict continuation_token:
+        The continuation token to retrieve the next page of results.
+    :ivar current_page:
+        The current page of listed results.
+    :vartype current_page: list(dict)
+
+    :param ~azure.storage.blob.ContainerClient:
```

Comment:
this can be blob.aio.ContainerClient too ;)

Reply:
good catch!